lib/rules/callback-return.js

```diff
@@ -52,7 +53,9 @@ module.exports = {
          * @returns {Boolean} Whether or not this function matches our callback name.
          */
         function isCallback(node) {
-            return node.callee.type === "Identifier" && callbacks.indexOf(node.callee.name) > -1;
+            return (node.callee.type === "Identifier" ||
+                (node.callee.type === "MemberExpression" && (node.callee.object.type === "Identifier" || node.callee.object.type === "MemberExpression")))
```

Comment:
This looks kind of gross but it allows us to check multiple levels deep while ignore cases like this: https://github.com/eslint/eslint/pull/6277/files#diff-f9620535a3e3dbfff1f0909369cbd8beR134